README.rst

```diff
@@ -367,6 +367,8 @@ Acknowledgements
 ----------------
 The development of this software was supported by funding to the Blue Brain Project, a research center of the École polytechnique fédérale de Lausanne (EPFL), from the Swiss government’s ETH Board of the Swiss Federal Institutes of Technology.
 
+This project/research received funding from the European Union’s Horizon 2020 Framework Programme for Research and Innovation under the Framework Partnership Agreement No. 650003 (HBP FPA).
```

Comment:
Where did you get that part ?

Reply:
NeuroM readme

Reply:
this is to acknowledge the full project period. That is not the case for libsonata. So we can scratch that one.

Reply:
ok